src/suites/cts/validation.spec.ts

```diff
@@ -0,0 +1,673 @@
+export const description = `
+Validation tests.
+`;
+
+import { TestGroup } from '../../framework/index.js';
+
+import { ValidationTest } from './validation_test.js';
+
+export const g = new TestGroup(ValidationTest);
+
+g.test('validation/binding count mismatch', async t => {
```

Comment:
no need for `validation/` at the beginning, since this file is already `validation.spec.ts`

Reply:
Done.